### ISSUE TITLE
Internal Server Error when resetting website

### ISSUE THREAD

**Issue by ov3rk1ll** [umami-software/umami#2915]
### Describe the Bug

I'm running the latest version in a docker environment.

When I try to reset a website, I only get a "Internal Server Error" on the page
![image](https://github.com/user-attachments/assets/e13e712e-c0a0-44bf-b268-8ee1ba6ce8a5)

as well as a prisma error on the container log

### Database

PostgreSQL

### Relevant log output

```shell
Invalid `prisma.session.deleteMany()` invocation:
The change you are trying to make would violate the required relation 'SessionToSessionData' between the `SessionData` and `Session` models.
    at _n.handleRequestError (/app/node_modules/@prisma/client/runtime/library.js:121:7749)
    at _n.handleAndLogRequestError (/app/node_modules/@prisma/client/runtime/library.js:121:7057)
    at _n.request (/app/node_modules/@prisma/client/runtime/library.js:121:6741)
    at async l (/app/node_modules/@prisma/client/runtime/library.js:130:9355) {
  code: 'P2014',
  clientVersion: '5.17.0',
{
    modelName: 'Session',
    relation_name: 'SessionToSessionData',
    model_a_name: 'SessionData',
    model_b_name: 'Session'
  }
}
```

### Which Umami version are you using? (if relevant)

v2.13.1

**Comment by franciscao633** [comment]
Forgot to add the cascading delete to prisma for `session_data`. If you need to reset the website in the meantime just delete all the data in that table for that specific website, then the reset should work.

**Comment by ThorANilsson** [comment]
> Forgot to add the cascading delete to prisma for `session_data`. If you need to reset the website in the meantime just delete all the data in that table for that specific website, then the reset should work.

This solution works.

**Comment by dmonterde** [comment]
Hi, this is also present on the following relation:

```
The change you are trying to make would violate the required relation 'EventDataToWebsiteEvent' between the `EventData` and `WebsiteEvent` models.
    at _n.handleRequestError (/app/node_modules/@prisma/client/runtime/library.js:121:7749)
    at _n.handleAndLogRequestError (/app/node_modules/@prisma/client/runtime/library.js:121:7057)
    at _n.request (/app/node_modules/@prisma/client/runtime/library.js:121:6741)
    at async l (/app/node_modules/@prisma/client/runtime/library.js:130:9355) {
  code: 'P2014',
  clientVersion: '5.17.0',
  meta: {
    modelName: 'WebsiteEvent',
    relation_name: 'EventDataToWebsiteEvent',
    model_a_name: 'EventData',
    model_b_name: 'WebsiteEvent'
  }
}
```

**Comment by mikecao** [comment]
Fixed in v2.14.0.
